src/Akka.Persistence.MongoDb.Hosting.Tests/MongoDbJournalOptionsSpec.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Akka.Configuration;
+using FluentAssertions;
+using FluentAssertions.Extensions;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Akka.Persistence.MongoDb.Hosting.Tests
+{
+    public class MongoDbJournalOptionsSpec
```

Comment:
These tests are basically clones of the ones here https://github.com/akkadotnet/Akka.Persistence.PostgreSql/blob/dev/src/Akka.Persistence.PostgreSql.Tests/Hosting/PostgreSqlOptionsSpec.cs

I have split them up and cleaned up parts but if there are additional cases needed I am happy to add them.